work.html

```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: NTIS
+---
+
+<div class="usa-grid">
+  <h1>Work with us</h1>
+
+    <h2>Building a new bridge between government and private sector innovation.</h2>
```

Comment:
If we're going to say this phrase - then we should explain what it means. How does it all work?